dcos/dcos-ui
# DCOS-17565: DC/OS Enterprise CLI description is wrong

It currently points to `dcos service`, it'd be more useful to show the
`dcos package install` command instead.

Closes https://jira.mesosphere.com/browse/DCOS-17565

## Testing
1. Go to the catalog
2. Find the `dcos-enterprise-cli` package and open it
3. Verify that the documentation link ends with `/dcos-package-install/`.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
(it's 1.11, because I am using my own cluster, it shouldn't matter)
### Before
![screenshot from 2019-01-22 10-35-18](https://user-images.githubusercontent.com/40791275/51524248-53880080-1e36-11e9-86e6-c78ae5c3dbdb.png)

### After
![screenshot from 2019-01-22 10-37-15](https://user-images.githubusercontent.com/40791275/51524259-597de180-1e36-11e9-84d2-ddcfb1a06697.png)
